Scripts/generate_processor_docs.py

```diff
@@ -167,7 +172,7 @@ def main(_):
     processor_heading = "  * **Processor Reference**"
     toc_string = ""
     toc_string += processor_heading + "\n"
-    for processor_name in processor_names():
+    for processor_name in sorted(processor_names()):
```

Comment:
And now I understand why this "broke". Ultimately the processor names come from a os.listdir() call. Under HFS+ those names were returned already sorted (in the "alphabetical" order one normally sees when listing a directory). But with APFS file systems, os.listdir() does not return a list of lexically-sorted file names.